MPCAutofill/MPCAutofill/.env.dist

```diff
@@ -37,3 +37,8 @@ SITE_NAME=MPC Autofill
 REDDIT=https://www.reddit.com/r/mpcproxies/
 DISCORD=http://mprox.link/discord
 THEME=superhero
+
+# PATREON - Key from "Creator's Access Token"
+PATREON_KEY=
+PATREON_URL=http://patreon.com/mpcfill
```

Comment:
let's not leave this url in the default env file or in `settings.py`

Reply:
Done